docs/core/tools/dotnet-nuget-trust.md

```diff
@@ -0,0 +1,319 @@
+dotnet nuget trust remove <name> [OPTIONS]
+```
+
+#### Arguments
+
+- **`name`**
+
+  The name of existing trusted signer going be removed.
+  
+#### OPTIONS:
+
+- **`-h|--help`**
+
+  Prints out a description of how to use the command.
+
+- **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+
+- **`--configfile <PATH>`**
+
+  Specific NuGet file that should be used instead of the standard hierarchy.
+
+### `author`
+
+Adds a trusted signer with the given name, based on the author signature of the package.
+
+#### Synopsis
+
+```dotnetcli
+dotnet nuget trust author <name> <package> [OPTIONS]
+```
+
+#### Arguments
+
+- **`name`**
+
+  The name of existing trusted signer going be added. If `name` already exists in the configuration, the signature will be appended.
+
+- **`package`**
+
+  The given `package` should be local path to the signed .nupkg file.
+  
+#### OPTIONS:
+
+- **`--allow-untrusted-root`**
+
+  Specifies if the certificate for the trusted signer should be allowed to chain to an untrusted root. This is not recommended.
+
+- **`-h|--help`**
+
+  Prints out a description of how to use the command.
+
+- **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+
+- **`--configfile <PATH>`**
+
+  Specific NuGet file that should be used instead of the standard hierarchy.
+
+### `repository`
+
+Adds a trusted signer with the given name, based on the repository signature or countersignature of a signed package.
+
+#### Synopsis
+
+```dotnetcli
+dotnet nuget trust repository <name> <package> [OPTIONS]
+```
+
+#### Arguments
+
+- **`name`**
+
+  The name of existing trusted signer going be added. If `name` already exists in the configuration, the signature will be appended.
+
+- **`package`**
+
+  The given `package` should be local path to the signed .nupkg file.
+
+#### OPTIONS:
+
+- **`--allow-untrusted-root`**
+
+  Specifies if the certificate for the trusted signer should be allowed to chain to an untrusted root. This is not recommended.
+
+- **`-h|--help`**
+
+  Prints out a description of how to use the command.
+
+- **`-v, --verbosity <LEVEL>`**
+
+  Set the verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+
+- **`--configfile <PATH>`**
+
+  Specific NuGet file that should be used instead of the standard hierarchy.
+
+### `certificate`
+
+Adds a trusted signer with the given name, based on a certificate fingerprint.
+
+#### Synopsis
+
+```dotnetcli
+dotnet nuget trust certificate <name> <fingerprint> [OPTIONS]
+```
+
+#### Arguments
+
+- **`name`**
+
+  The name of existing trusted signer going be added. If a trusted signer with the given name already exists, the certificate item will be added to that signer. Otherwise a trusted author will be created with a certificate item from given certificate information.
```

Comment:
```suggestion
  The name of the existing trusted signer to add. If a trusted signer with the given name already exists, the certificate item is added to that signer. Otherwise a trusted author is created with a certificate item from the given certificate information.
```